src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -110,19 +147,22 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
       <PageTitle title="New Change Request" previousPages={[{ name: 'Change Requests', route: routes.CHANGE_REQUESTS }]} />
       <PageBlock title="Details">
         <Grid container spacing={2}>
-          <Grid item xs={12} md={3}>
+          <Grid item xs={12} md={6}>
             <Box>
-              <Typography variant="caption">WBS Number</Typography>
+              <Typography sx={{ ml: 1 }} variant="caption">
+                WBS
+              </Typography>
             </Box>
-            <ReactHookTextField
-              name="wbsNum"
-              control={control}
-              placeholder="1.1.0"
-              errorMessage={errors.wbsNum}
-              sx={style}
+            <NERAutocomplete
+              id="wbs-autocomplete"
+              onChange={wbsAutocompleteOnChange}
+              options={wbsDropdownOptions}
+              size="small"
+              placeholder="Select a project or work package"
+              value={wbsDropdownOptions.find((element) => element.id === wbsNum) || null}
             />
           </Grid>
-          <Grid item xs={12} md={9}>
+          <Grid item xs={12} md={3}>
             <Box>
               <Typography variant="caption">Type</Typography>
             </Box>
```

Comment:
I'd say put type on a new line on its own. Looks a lil goofy, but better than having it right next to wbs like it is now imo

Reply:
agree